libbeat/esclientleg/json_read.go

```diff
@@ -15,7 +15,7 @@
 // specific language governing permissions and limitations
 // under the License.
 
-package elasticsearch
+package esclientleg
```

Comment:
this file was introduced specifically for the elasticsearch output only. If possible we should keep it and related types in the elasticsearch output (but unexported). The less symbols/functionality we provide in the esclientleg package, the better. What might be somewhat confusing is that the client has different APIs for bulk requests, all working a little different.

Reply:
Fixed in 08e4c6f9bf6f5a5106263c9ac07b01c6e70470a9.